spec/factories.rb

```diff
@@ -424,7 +425,7 @@ def r_str
   factory :auth_with_read_and_write, class: Api::OpenidConnect::Authorization do
     o_auth_application
     user
-    scopes %w(openid sub aud profile picture nickname name read write)
+    scopes { %w(openid sub aud profile picture nickname name read write) }
```

Comment:
Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`.